src/components/Personality.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import {
+  Image,
+  Platform,
+  ScrollView,
+  StyleSheet,
+  Slider,
+  Text,
+  TouchableOpacity,
+  TouchableHighlight,
+  View,
+} from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+
+class Personality extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      values: [],
+      sliderTitle: [
+        ['Extroverts', 'Introverts'],
+        ['Sensors', 'Intuitives'],
+        ['Feelers', 'Thinkers'],
+        ['Judgers', 'Percievers'],
+        ['Speaker', 'Listener'],
+        ['Energetic', 'Quite'],
+        ['Unexacting', 'Perfectionist'],
+        ['Traditional', 'Free Thinking'],
+      ],
+    };
+  }
+
+  render() {
+
+    let slidersWithTitle = this.state.sliderTitle.map((currentValue, index) => (
+        <View key={index}>
+          <View style={styles.sliderTitle}>
+            <Text style={{ color: '#757b7c' }}>{currentValue[0]}</Text>
+            <Text style={{ color: '#757b7c' }}>{currentValue[1]}</Text>
+          </View>
+          <Slider
+            style={styles.slider}
+            onValueChange={(value) => {
+              let newValues = this.state.values.slice();
+              newValues[index] = value;
+              this.setState({ values: newValues });
+            }}
+
+            maximumValue={4}
+            minimumValue={0}
+            step={1}
+          />
+        </View>
+      )
+    );
+    return (
+      <ScrollView style={styles.scroll}>
+        <View style={styles.titleContainer}>
+          <Text style={styles.title}>Let’s figure out your personality !</Text>
```

Comment:
ohhh so the space you added in the code actually makes that space in the design guide as well? I did not know it.